SpotMenu.xcodeproj/project.pbxproj

```diff
@@ -382,7 +378,7 @@
 					};
 					64185C0B1F1B8DC400CA23A0 = {
 						CreatedOnToolsVersion = 8.3.3;
-						DevelopmentTeam = 3K79XCJYWL;
+						DevelopmentTeam = F6W8CTW858;
```

Comment:
Thanks 🙌🏻 Everything looks good to me, but could you change the `DevelopmentTeam` back to its original value before I merge?